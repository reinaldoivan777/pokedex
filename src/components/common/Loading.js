import React from "react";
import { Spinner } from "react-bootstrap";

const Loading = () => (
  <Spinner animation="border" role="status" className="my-3">
    <span className="sr-only">Loading...</span>
  </Spinner>
);

export default Loading;
