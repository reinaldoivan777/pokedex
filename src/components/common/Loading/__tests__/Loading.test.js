import React from "react";
import renderer from "react-test-renderer";

import Loading from "../index";

describe("Loading", () => {
  it("render without error", () => {
    renderer.create(<Loading />);
  });

  it("snapshot renders", () => {
    const component = renderer.create(<Loading />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
