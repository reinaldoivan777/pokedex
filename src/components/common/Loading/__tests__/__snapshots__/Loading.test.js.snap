// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading snapshot renders 1`] = `
<div
  className="my-3 spinner-border"
  role="status"
>
  <span
    className="sr-only"
  >
    Loading...
  </span>
</div>
`;
